Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView.swift

```diff
@@ -5,63 +5,34 @@
 import StreamChat
 import UIKit
 
+/// A view that shows a number of unread messages in channel.
 public typealias ChatChannelUnreadCountView = _ChatChannelUnreadCountView<NoExtraData>
-
+/// A view that shows a number of unread messages in channel.
 open class _ChatChannelUnreadCountView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
-    // MARK: - Properties
+    /// The `UILabel` instance that holds number of unread messages.
+    open private(set) lazy var unreadCountLabel = UILabel().withoutAutoresizingMaskConstraints
```

Comment:
this should probably also have `adjustingFontForContentSizeCategory` -> https://github.com/GetStream/stream-chat-swift/pull/819

Reply:
It was before audit already (line 43) 🙂